net/Sample/Views/Home/GroupedList.cshtml

```diff
@@ -10,14 +10,14 @@
                 store: DevExpress.data.AspNet.createStore({
                     loadUrl: "@Url.Action("Products", "Northwind")"
                 }),
-                group: "Category.CategoryName",
-                sort: "ProductName",
+                group: "category.categoryName",
```

Comment:
@AlekseyMartynov 

Regarding [this](https://github.com/aspnet/Announcements/issues/194) breaking change, EF serializes JSON in this way. 

We have 2 options to fix this:
1) change letter case;
2) restore previous naming strategy using follow code:

     services
         .AddMvc()
         .AddJsonOptions(options => options.SerializerSettings.ContractResolver = new DefaultContractResolver());

I chose 1st option.